### ISSUE TITLE
Add missing layout in C4_Deployment Diagram Sample - bigbankplc.puml

### ISSUE THREAD

**Issue by kirchsth** [plantuml-stdlib/C4-PlantUML#92]
@adrianvlupu thank you that you merged my dynamic and deployment extension into stdlib.

In the "C4_Deployment Diagram Sample - bigbankplc.puml" I would (re)add 
- the directed relations
```
Rel(mobile, api, "Makes API calls to", "json/HTTPS")
Rel(spa, api, "Makes API calls to", "json/HTTPS")
Rel_U(web, spa, "Delivers to the customer's web browser")
Rel(api, db, "Reads from and writes to", "JDBC")
Rel(api, db2, "Reads from and writes to", "JDBC")
Rel_R(db, db2, "Replicates data to")
```

- add the linebreak in the node tech (I fixed it<size:$TECHN_FONT_SIZE>)
```
Deployment_Node(comp, "Customer's computer", "Mircosoft Windows of Apple macOS"){
    Deployment_Node(browser, "Web Browser", "Google Chrome, Mozilla </size>\n<size:$TECHN_FONT_SIZE>Firefox, Apple Safari or </size>\n<size:$TECHN_FONT_SIZE>Microsoft Edge"){
        Container(spa, "Single Page Application", "JavaScript and Angular", "Provides all of the Internet Banking functionality to customers via their web browser.")
    }
}
```

And in the "README" I would 
- add "$index" as dynamic macro

Should I do it?

Best regards
Helmut

**Comment by adrianvlupu** [comment]
Hi and thanks for the contribution.
Sure you can change the sample, but it would be nicer if we could find a version where adding a `\n` in the type parameter wouldn't break the layout.
Tried a couple of fixes already but nothing seemed to work

**Comment by kirchsth** [comment]
If we write it with "</size>\n<size:$TECHN_FONT_SIZE>" (new with $ based on the different variable definition) then the format would work again

PS.: In the meantime I started with a version which makes an automatic break on each space, but I cannot extract the implementation in a separate function (atm. it has ~30 lines of code and I want to extract it from the Deployment_Node)

**Comment by kirchsth** [comment]
fixed, changes merged with PR #107
